### ISSUE TITLE
Node version glitch on homebridge-hue 0.13.64

### ISSUE THREAD

**Issue by jostrasser** [ebaauw/homebridge-hue#1163]
#### Issue
Node.js version checking glitch in the log after updating

#### Log Messages
```
[22/07/2023, 22:54:16] [Homebridge-Hue] warning: recommended version: node vundefined LTS
```

Installed Node.js Version | v18.17.0

**Comment by ebaauw** [comment]
Damn, forgot to change this for my old, static platform accessory plugins.

**Comment by ebaauw** [comment]
FIxed in v0.13.65,
